examples/jf-control-app/commands/pairing/PairingCommand.cpp

```diff
@@ -28,6 +28,7 @@
 #include <crypto/CHIPCryptoPAL.h>
 #include <lib/core/CHIPSafeCasts.h>
 #include <lib/dnssd/Advertiser.h>
+#include <lib/support/RpcErrorMapping.h>
```

Comment:
RpcErrorMapping does not appear to be used in this module.  Please remove if not in use.  The only change to this module is changing the CHIP_ERROR_SHUT_DOWN -> CHIP_ERROR_INTERNAL